packages/icheck-ts/src/cli.ts

```diff
@@ -39,7 +47,12 @@ const command = new TinyCliCommand(
       file: args.cacheLocation,
     });
     await cacheOp?.load();
-    const result = await runner(args.files, { parse });
+    const result = await runner(args.files, {
+      parse,
+      experimental: process.execArgv.includes(
+        "--experimental-import-meta-resolve"
+      ),
```

Comment:
`process.execArgv` check is a little clumsy for cli usage...

```sh
tsx --experimental-import-meta-resolve ./node_modules/@hiogawa/icheck-ts/bin/cli.js
```

From https://github.com/hi-ogawa/ytsub-v3/blob/e0fa9d7f3ff47d458e4ea77693213e640565936d/package.json#L17

Reply:
Now this is changed to explicit `--useImportMetaResolve` flag https://github.com/hi-ogawa/js-utils/pull/184/commits/a0e1f32596699035fc89218a320bd07557f803ab.

It still looks funny but I guess it's better than not being able to use `NODE_OPTIONS`:

```sh
NODE_OPTIONS='--import tsx/esm --experimental-import-meta-resolve' icheck-ts --useImportMetaResolve
```

from https://github.com/hi-ogawa/ytsub-v3/pull/546